### DISCUSSION TITLE
How do I create a new project?

### DISCUSSION THREAD

**Discussion by jrcloutier** [ICIJ/datashare#1027] [Q&A] [answered]
On the Datashare [demo page](https://datashare-demo.icij.org/?signout#/), there's a list of projects under the search bar. How do I create a new project? Is this option only available for datashare running in server mode?

**Comment by pirhoo** [comment, marked as answer]
Hello @jrcloutier, 

the demo page is in "SERVER" mode which allow multi-projects (and multi-users) in Datashare :

https://github.com/ICIJ/datashare/wiki/Datashare-Server-Mode

The installation of this mode is more complex but brings more features. Luckily, on our roadmad we want to bring multi-project to the "LOCAL" mode of Datashare.

**Reply by jrcloutier** [reply]
That's what I suspected. Thanks for confirming @pirhoo.
